mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.erb

```diff
@@ -27,6 +27,7 @@ func TestAccContainerCluster_basic(t *testing.T) {
 				Check: resource.ComposeAggregateTestCheckFunc(
 					resource.TestCheckResourceAttrSet("google_container_cluster.primary", "services_ipv4_cidr"),
 					resource.TestCheckResourceAttrSet("google_container_cluster.primary", "self_link"),
+					resource.TestCheckResourceAttr("google_container_cluster.primary", "networking_mode", "VPC_NATIVE"),
```

Comment:
Simple test to ensure the customize diff default is working.
Regular upgrade testing was done manually to ensure the upgrade did not affect existing routes-based clusters.